extensions-core/lookups-cached-global/src/main/java/io/druid/query/lookup/namespace/URIExtractionNamespace.java

```diff
@@ -530,8 +531,10 @@ public JSONFlatDataParser(
       Preconditions.checkArgument(!Strings.isNullOrEmpty(valueFieldName), "[valueFieldName] cannot be empty");
       this.keyFieldName = keyFieldName;
       this.valueFieldName = valueFieldName;
+
+      // Copy jsonMapper; don't want to share canonicalization tables, etc., with the global ObjectMapper.
       this.parser = new DelegateParser(
-          new JSONParser(jsonMapper, ImmutableList.of(keyFieldName, valueFieldName)),
+          new JSONParser(jsonMapper.copy(), ImmutableList.of(keyFieldName, valueFieldName)),
```

Comment:
Do you remember what kind of bugs?

If it's things along the lines of "not all configs and modules are copied" then that should be fine -- the jsonMapper shouldn't be finicky to that, since it's just reading generic `Map<String, String>` and not serializing anything.